configs/autopush_shared.ini.sample

```diff
@@ -125,7 +125,8 @@ endpoint_port = 8082
 ;        "topic": "com.mozilla.org.Firefox",  // Bundle ID for associated app
 ;        "max_connections": 100, // Max number of connection pool entries.
 ;        "sandbox": False},  // Use the APNs sandbox feature
+         "max_retry": 2, // Max number of retries in event of an HTTP2 error
```

Comment:
So going by line 130 it's a knob specific to the rel_channel. Then the '}' on the line above should move down to this line.

But in that case it wouldn't be `router_conf.get('max_retry')`, but `router_conf[rel_channel].get('max_retry')`. That's what I meant by potentially clashing rel_names at the top level

Reply:
ah, no, that's just a typo in the comment. It's per platform so in theory `router_conf[rel_channel].get('max_retry')`